### ISSUE TITLE
Not able to subscribe even when made public

### ISSUE THREAD

**Issue by perrodecroy** [skandragon/thing#147]
I've tried a couple of times now to subscribe to my personal link. I've made the calendar public but when I subscribe to the link nothing comes up on the calendar. I'm using google through Chrome and the link I try to subscribe to is: http://thing.pennsicuniversity.org/users/1816/schedule.ics 

Thanks for your help!

**Comment by johnjmikucki** [comment]
I see a similar issue when attempting to subscribe from from google calendar as well as a third-party calendaring app.  Visiting the URL downloads what appears to be a properly-formatted ICS file with contents for courses I have selected.  The file appears (~4800 lines, I haven't 100% manually verified it) to import successfully into the third-party calendaring app.

Possibly related: some class descriptions appear to have embedded HTML or other formatting which isn't rendering correctly in my browser.  If I can get better diagnostic info I'll post here.

**Comment by johnjmikucki** [comment]
Well that's interesting.
attempting to curl the http://thing.pennsicuniversity.org/users/3548/schedule.ics URL returns a 404.  Banging the ICS URL against an ical validator also throws a couple errors (it gets a 404 as well).  Report here:
https://icalendar.org/validator.html?url=http://thing.pennsicuniversity.org/users/3548/schedule.ics

The content of the .ics downloaded by my browser, pasted manually into the validator, passes with flying colors--no errors or warnings.  This plus the successful import makes me think it could be a mismatch between what (the URL, headers, etc) Thing expects/handles and what it is receiving from would-be subscribers.  If I had to guess blindly I'd check headers, content types, something like that.  Hoping there might be something helpful in the Thing's server logs...

Happy to dig further if I can be of help!

**Comment by perrodecroy** [comment]
Yeah, according to the "how to" I should be getting a string of alpha numeric after the ...users/ bit but all I have is 4 digits. Could that have something to do with it?
